### DISCUSSION TITLE
About Installation

### DISCUSSION THREAD

**Discussion by CloudZhang-6** [deepmodeling/deepmd-kit#1454] [Q&A]
i'm installing dp kit with conda using:
conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps-dp cudatoolkit=11.3 horovod -c https://conda.deepmodeling.org
Here is the error message i got

> $ conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps-dp cudatoolkit=11.3 horovod -c https://conda.deepmodeling.org
> Fetching package metadata ...
> CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.deepmodeling.org/linux-64/repodata.json>
> Elapsed: -
> 
> An HTTP error occurred when trying to retrieve this URL.
> HTTP errors are often intermittent, and a simple retry will get you on your way.
> SSLError(SSLError(SSLError("bad handshake: Error([('SSL routines', 'ssl3_get_server_certificate', 'certificate verify failed')],)",),),)
> 

I also tried to open the webpage https://conda.deepmodeling.org, it says 
{
  "code": "not_found",
  "message": "File with such name does not exist.",
  "status": 404
}

I was wondering how to fix this probelm.
Thanks!

**Comment by njzjz** [comment]
I cannot reproduce the error. You may check your Internet.
